apply/v2/applydriver/local.go

```diff
@@ -171,10 +177,7 @@ func (c *Applier) scaleCluster(mj, md, nj, nd []string) error {
 
 func (c *Applier) upgradeCluster(mj, nj []string) error {
 	// use k8sClient to fetch current cluster version.
-	info, err := c.Client.GetClusterVersion()
-	if err != nil {
-		return err
-	}
+	info := c.CurrentClusterInfo
```

Comment:
Where is c.CurrentClusterInfo been set?

Reply:
start from reconcileCluster() ,we set this info firstly. line 109